docs/usage.rst

```diff
@@ -34,9 +34,27 @@ Non-blocking lock::
     else:
         print("Someone else has the lock.")
 
-Controlled execution lock::
+Releasing previously acquired lock::
+
+    conn = StrictRedis()
+    lock = redis_lock.Lock(conn, "name-of-the-lock")
+    lock.acquire()
+    print("Got the lock. Doing some work ...")
+    time.sleep(5)
+    lock.release()
+
+The above example could be rewritten using context manager::
 
     conn = StrictRedis()
     with redis_lock.Lock(conn, "name-of-the-lock"):
         print("Got the lock. Doing some work ...")
         time.sleep(5)
+
+If lock was acquired in other context (other process, remote worker server,
+etc), it's still possible to release it with parameter `force` equal
+to ``True``::
```

Comment:
Hmmm, oops. We need to fix `reset()` then ...

The idea is that each Lock instance (the python object) has a unique id and that is used to prevent "accidental" releases from wrong places (that don't have the same unique id as whoever acquired)